src/preset_cli/auth/superset.py

```diff
@@ -28,24 +27,43 @@ def __init__(self, baseurl: URL, username: str, password: Optional[str] = None):
     def get_headers(self) -> Dict[str, str]:
         return {"X-CSRFToken": self.csrf_token} if self.csrf_token else {}
 
+    def get_access_token(self):
+        """
+        Get an access token from superset API: api/v1/security/login.
+        """
+        body = {
+            "username": self.username,
+            "password": self.password,
+            "provider": "ldap",
```

Comment:
Would `ldap` work in all cases? I'm not too familiar with the provider variances in Superset, but I know that on my end when testing I set it to `db`. Wondering if this should be a dynamic config.

Reply:
Good point. I think it's better to change to dynamic config. Let me change it.